hack/cleanup/gke_clusters.go

```diff
@@ -20,7 +20,7 @@ const timeUntilClusterOrphaned = time.Hour
 func cleanupGKEClusters(ctx context.Context, log logr.Logger) error {
 	var creds map[string]string
 	if err := json.Unmarshal([]byte(gkeCreds), &creds); err != nil {
-		return fmt.Errorf("invalid credentials: %w", err)
+		return fmt.Errorf("Invalid credentials: %w", err)
```

Comment:
Do we want to capitalize errors? Popular Go style guides don't recommend it
- [Go recommendation](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)
- [Google](https://google.github.io/styleguide/go/decisions.html#error-strings)
- stdlib, etc. don't do it too

🤔

Reply:
No we don't :) https://github.com/Kong/kubernetes-ingress-controller/pull/5033#discussion_r1377792342